### PR TITLE
Node - Week3 Assignment

### PR DESCRIPTION
**Pull request by AnnaSolovykh**

### CODE REVIEW

File: 02-express-tutorial/app.js
Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: Static routing well implimented. Unmatched request (Not found well implimented)

**Review by leo-mutuku — APPROVED** [review]

File: 02-express-tutorial/app.js
Comment (leo-mutuku) [inline 02-express-tutorial/app.js:1]: Returing response  over http. Well implimented

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]
Returing ptroducts from a data file. well implimented

File: 02-express-tutorial/public/index.html
Comment (leo-mutuku) [inline 02-express-tutorial/public/index.html:1]: api endpoints well linked on static page

**Review by leo-mutuku — APPROVED** [review]

**Review by leo-mutuku — APPROVED** [review]
This is perfect! Destructuring in practice

**Review by leo-mutuku — APPROVED** [review]
Request params well implimented!

**Review by leo-mutuku — APPROVED** [review]
Finding single product well implimented

**Review by leo-mutuku — APPROVED** [review]
Search and limit queries well implimented

**Review by leo-mutuku — APPROVED** [review]
search filter sort with regex well implimented

**Review by leo-mutuku — APPROVED** [review]
maximum price filter well implimented. Perfect use of the spread syntax

**Review by leo-mutuku — APPROVED** [review]
Great work! Refactoring to make code more readable.

**Review by leo-mutuku — APPROVED** [review]
Product fetching on click of a button well implimented.

**Review by leo-mutuku — APPROVED** [review]
I like the game! Clean code easy to read.

**Comment by leo-mutuku** [comment]
Great work. Right from simple and well describe commit messages to refactoring. Carry on.

**Comment by AnnaSolovykh** [comment]
> Great work. Right from simple and well describe commit messages to refactoring. Carry on.

Happy to hear that! 
Thank you for the review!

**Pull request merged**